server/events.go

```diff
@@ -181,6 +181,7 @@ type ClientInfo struct {
 	RTT        time.Duration `json:"rtt,omitempty"`
 	Server     string        `json:"server,omitempty"`
 	Cluster    string        `json:"cluster,omitempty"`
+	Alternates []string      `json:"alts,omitempty"`
```

Comment:
`alts` is a bit unclear, hard to guess the meaning for someone who dont know the code

Reply:
I think you will need sample code or docs for this. I tried other names to make it a bit more clear but that did not seem to help much tbh.

Reply:
ClientInfo has not been documented formally yet either. We have been watching and adapting its use for JetStream, even though it is a general construct across service export boundaries.